File: 110algoritmos/exercicios006/ex006-Csharp/ex006.cs
using System;

class Program {
    static void Main() {
        int num, dobro, triplo;
        double raiz;

        Console.Write("Digite um número: ");
        num = int.Parse(Console.ReadLine());

        dobro = num * 2;
        triplo = num * 3;
        raiz = Math.Pow(num, 0.5);

        Console.WriteLine("Do dobro de " + num + " é " + dobro);
        Console.WriteLine("Do triplo é " + triplo);
        Console.WriteLine("A raiz é " + raiz.ToString("f"));

    }
}

File: 110algoritmos/exercicios006/ex006-Go/ex006.go
package main

import (
	"fmt"
	"math"
)

func main() {

	var num float64

	fmt.Printf("Digite um número: ")
	fmt.Scan(&num)

	dobro := num * 2
	triplo := num * 3
	raiz := math.Pow(num, 0.5)

	fmt.Printf("O dobro de %v é %v \nO tripo é: %v. \nA raiz é: %.2f", num, dobro, triplo, raiz)

}
